mandatory/2-function-creation.js

```diff
@@ -15,15 +20,24 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
+function validate(num) {
+  if (typeof num === "number" && num %2 === 0 && num <= 100) {
```

Comment:
The condition ``typeof num === "number" && num %2 === 0 && num <= 100`` already returns true/false, so you don't need to use an if statement anymore for this. Would you be able to correct this? It will simplify your code and make it easier to read